artemis-pom/pom.xml

```diff
@@ -0,0 +1,870 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Similarly, this new file has Windows line endings, switch it to Unix for consistency with others.